xbmc/Application.cpp

```diff
@@ -2939,6 +2939,42 @@ void CApplication::Stop(int exitCode)
   Sleep(200);
 }
 
+bool CApplication::ResumePlayer(const CFileItemPtr& itemIn) {
+  if (itemIn->IsEmpty())
+  {
+    CLog::Log(LOGDEBUG, "CApplication::ResumePlayer nothing to resume");
+    return false;
+  }
+
+  CLog::Log(LOGDEBUG, "CApplication::ResumePlayer resume item %s", itemIn->GetPath().c_str());
+  if (m_pPlayer->HasPlayer())
+  {
+    CLog::Log(LOGWARNING, "CApplication::ResumePlayer can't resume, player is already startet");
+    return false;
+  }
+
+  CFileItem item(itemIn->GetItemToPlay());
+  CLog::Log(LOGDEBUG, "CApplication::ResumePlayer resume item to play %s", item.GetPath().c_str());
+#ifdef HAS_DVD_DRIVE
+  if (item.IsDVD() || item.IsCDDA())
+    return MEDIA_DETECT::CAutorun::PlayDisc(item.GetPath(), true, false);
```

Comment:
I would like to remove this because PlayDisc does just create a usable media path. And we do already have the resolved path in memory during onwake.

But I'm not sure about RunDisc https://github.com/xbmc/xbmc/blob/8ddffac5c558ef679c191ebc0207e84397a7ace9/xbmc/Autorun.cpp#L137 which gets called by PlayDisc
I guess we do not store enoght of the player state to resume with play media instead of RunDisc